flashcards/backend/decks/models.py

```diff
@@ -20,3 +22,40 @@ def __str__(self):
     class Meta:
         verbose_name = "category"
         verbose_name_plural = "categories"
+
+
+class Tag(models.Model):
+    """Model for representing tags for decks
+    :param name: Tag name
+    :type name: str
+    :param description: Description for tag
+    :type description: str
+    :param deck: Foreign Key for Deck model
+    :type deck: int
+    :param flashcard: Foreign Key for Flashcard model
+    :type flashcard: int
+    """
+
+    name = models.CharField(max_length=24)
+    description = models.TextField()
```

Comment:
do we need a description for a tag? And if so I really would cut it out from admin panel.

Reply:
Right, I remove tag description from model and from test.